lib/runner/variant-reassignment.js

```diff
@@ -266,6 +269,20 @@ export default class VariantReassignment {
     )
   }
 
+  /**
+   * In productDrafts from external systems, there's no productTypeId, but instead the key is used.
+   * However, productTypeId is needed for reassignment. This method replaces the productTypeKey
```

Comment:
Just for the record - current sphere-product-import resolves types by product-type name and not key.

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/56/commits/d98dcdce165c7726ffc7c0fede1c3a586aa4cf8b